### ISSUE TITLE
Cosigner pool broken - can fix, but server down too.

### ISSUE THREAD

**Issue by neocogent** [spesmilo/electrum#2084]
Cosignor pool seems like a great idea but this was my first time trying it out. But right now the "Send to Cosignor" button does not appear even if plugin is enabled.

Looks like line 133 in cosignor/qt.py is a relic of an old version:
`if wallet.master_private_keys.get(key):`

I tried updating this to use keystore:
`if not keystore.is_watching_only():`

This allowed button to show and when clicked the problem is that now server is refusing connections. So I'd guess it's not up - probably due to other broken code issues. It appears that this plugin also needs fixing in the receive section - to use keystore to decode tx data. I haven't fixed at this yet but I could update this as well. It appears receive code needs to use the 'x%d/' path on keystore.decrypt_message instead of the old method.

Should I fix this and submit a PR or is this plugin being discontinued or otherwise no longer recommended?

**Comment by ecdsa** [comment]
fixed in e18e323c4579198a8910bc697a62fc2e0cec7ebb
I restarted the service on cosigner.electrum.org, it will take a while until the DNS is propagated.

**Comment by neocogent** [comment]
Thanks. I actually went ahead and fixed it here - testing with a tx using cosignorpool on my localhost. _Thought_ I was being helpful but you were quicker than me and I went to issue a PR and it was already done. I had a look at code you committed and turns out almost identical except I used keystore[key] instead of just keystore for getting xprv. I presume the first one will always match the incoming tx. Anyway, very cool.
